src/System.Data.Common/tests/System/Data/Common/DbDataReaderTest.netcoreapp.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Data.Common;
```

Comment:
Is this needed?

Reply:
@maryamariyan, yes, otherwise the extension methods cannot be resolved; i.e., the tests will only see the instance methods; the methods that take an `int`.